src/modules/deleteItem.js

```diff
@@ -3,12 +3,19 @@ import listStore from './listStore.js';
 
 const deleteItem = (id) => {
   const deleteButton = document.getElementById(`btn-${id}`);
-
-  const returnedItems = retrieveFromStore().filter((item) => item.index !== id);
+  const returnedItems = retrieveFromStore();
+  if (returnedItems) {
+    returnedItems.filter((item) => item.index !== id);
+    //  update the indices of the remaining items
+    returnedItems.forEach((item, index) => {
+      item.index = index;
+    });
+  }
+  // const returnedItems = retrieveFromStore().filter((item) => item.index !== id);
   // update the indices of the remaining items
-  returnedItems.forEach((item, index) => {
-    item.index = index;
-  });
+  // returnedItems.forEach((item, index) => {
+  //   item.index = index;
+  // });
```

Comment:
- [ ] Please don't forget to remove old pieces of commented code, they make your code base look ugly and make it less readable. 🤓 🔧